frontend/components/Archives.tsx

```diff
@@ -188,23 +241,47 @@ const Archives: React.FC<ArchivesProps> = ({
           </Text>
         )}
         <Flex alignItems={"center"} gap={1}>
-          <Link
-            isExternal={true}
-            href={
-              archive.ownerAddress
-                ? `${networkConfig.marketplaceProfilesURL}/${archive.ownerAddress}`
-                : undefined
-            }
-          >
-            <Avatar
-              name={archive.ownerName ? archive.ownerName : undefined}
-              src={archive.ownerAvatar ? archive.ownerAvatar : undefined}
-              height={"24px"}
-              width={"24px"}
-            />
-          </Link>
+          {isMintable(archive) && (
```

Comment:
I feel this component has outgrown its purpose because it's trying to cater for different data that is fetched from 2 different sources and has 2 different views

I commented on parent branch which may effect this implementation https://github.com/yearone-io/burntpix-archives/pull/59#discussion_r1560063289